packages/syft/src/syft/core/node/new/task/task_service.py

```diff
@@ -0,0 +1,428 @@
+        if task_owner in enclave_task.owners:
+            if enclave_task.status[task_owner] in ["Approved", "Denied"]:
+                return Err(
+                    SyftSuccess(
+                        message=f"Cannot Modify the status of task: {task.id} which has been Approved/Denied \n"
+                        + "Kindly Submit a new request"
+                    )
+                )
+            enclave_task.status[task_owner] = task.status[task_owner]
+            # Create nested action store of DO for storing intermediate data
+            if not self.action_store.exists(task_owner.node_uid):
+                dict_object = DictObject()
+                dict_object.base_dict[task.id] = private_input_map
+                self.action_store.set(
+                    uid=task_owner.node_uid,
+                    credentials=context.credentials,
+                    syft_object=dict_object,
+                )
+            else:
+                result = self.action_store.get(
+                    uid=task_owner.node_uid, credentials=context.credentials
+                )
+                if result.is_ok():
+                    result = result.ok()
+                    result.base_dict[task.id] = private_input_map
+                    self.action_store.set(
+                        uid=task_owner.node_uid,
+                        credentials=context.credentials,
+                        syft_object=result,
+                    )
+                else:
+                    return result.err()
+
+            # Check if all the DO approved the code execution
+            code_status_check = set(enclave_task.status.values())
+            if len(code_status_check) == 1 and "Approved" in code_status_check:
+                enclave_task.execution = "executing"
+                self.task_stash.update(enclave_task)
+                # TODO 🟣 Branch to separate thread for execution of enclave task
+                self.execute_task(enclave_task=enclave_task, context=context)
+
+            self.task_stash.update(enclave_task)
+
+            return Ok(True)
+        else:
+            return Err(
+                f"Task: {task.id.no_dash}:{enclave_task.owners} in enclave does not contain {task.owners[0].name} "
+            )
+
+    def fetch_private_inputs(
+        self, enclave_task: Task, context: AuthedServiceContext
+    ) -> Dict:
+        inputs = {}
+        for domain in enclave_task.owners:
+            domain_input = self.action_store.get(
+                uid=domain.node_uid,
+                credentials=context.credentials,
+                skip_permission=True,
+            )
+            if domain_input.is_ok():
+                inputs.update(domain_input.ok().base_dict[enclave_task.id])
+            else:
+                return domain_input.err()
+        return inputs
+
+    def execute_task(self, enclave_task: Task, context: AuthedServiceContext) -> None:
+        global stdout_
+        global stderr_
```

Comment:
`stdout_` and `stderr_` are only used in this function. We can just put

```py
stdout_ = sys.stdout
stderr_ = sys.etderr
```

here instead of using `global`? Similar to 43746afcb1498edb175d31a6a014a904f6874b20

Reply:
@kiendang , Initially , we implemented the old task service system.
Madhava suggested on porting to the new user code service..
This code would soon be deleted
Since the user code service was being developed parallely, 
We thought we could merge this PR and work on  porting to the new user code service in a new task/PR...

Reply:
I see. That explains why I see so much duplicated code with the code submission api.